bin/exec_demo_run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+source /software/lsstsw/stack/loadLSST.bash
+setup lsst_distrib
+setup obs_decam
+
+AP_VERIFY_HITS2015_DIR=/project/krzys001/ap_verify_hits2015
+setup -k -r $AP_VERIFY_HITS2015_DIR
+
+
+BASE_DIR=./
+setup -k -r $BASE_DIR/ap_association
+setup -k -r $BASE_DIR/ap_pipe
+setup -k -r $BASE_DIR/ap_verify
+
+
+
+# CCD 1 has image subtraction problems, CCDs 2 and 61 are missing
+for i in `seq 3 60`; do
+	ap_verify.py --dataset HiTS2015 --output ap_verify_HiTS --dataIdString "visit=${1} ccdnum=$i filter=g" --silent;
```

Comment:
`ap_verify.py` is not executable; it needs to be `python ap_verify.py`. Also, shouldn't it be `$BASE_DIR/ap_verify/bin/ap_verify.py`?